pkg/openthread/include/ot.h

```diff
@@ -17,7 +17,7 @@
  *
  * @file
  *
- * @author  Jos� Ignacio Alamos <jialamos@uc.cl>
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
```

Comment:
?

Reply:
the `é` of José looks weird when viewed in Atom or VIM editors, obviously it doesn't on github.